src/BulletDynamics/Featherstone/btMultiBodyDynamicsWorld.cpp

```diff
@@ -615,6 +616,7 @@ void btMultiBodyDynamicsWorld::integrateMultiBodyTransforms(btScalar timeStep)
 			{
 				bod->clearVelocities();
 			}
+            bod->substractSplitV();
```

Comment:
same, move within the sleep check, if possible.

Reply:
Done.